### PR TITLE
(WIP) Added 'To, Cc and Bcc' fields for mailing an invoice

### PR DESCRIPTION
**Pull request by mfrausto-mdw**
Overview
----------------------------------------
Added functionality and altered templates to add 'To, Cc and Bcc' fields for mailing an invoice

Before
----------------------------------------
mailing_invoice_before

After
----------------------------------------
mailing_invoice_after

Technical Details
----------------------------------------
-Added toEmails property to `CRM/Contact/Form/Task/EmailCommon.php`
-Modified ` CRM/Contribute/Form/Task/Invoice.php` file in order to add the new form elements and rules for them
-Added helper function into `CRM/Core/BAO/Email.php`
-Added help assets to `templates/CRM/Contact/Form/Task/Email.hlp`
-Modified template ` templates/CRM/Contribute/Form/Task/Invoice.tpl` in order to show fields added

Comments
----------------------------------------
https://mydropwizard.atlassian.net/browse/ROUND-669

### CODE REVIEW

File: CRM/Contribute/Form/Task/Invoice.php [hunk 1]
```diff
@@ -156,6 +159,12 @@ public function buildQuickForm() {
     }
     else {
       $this->addRule('from_email_address', ts('From Email Address is required'), 'required');
+      $this->addRule('to_email_address', ts('To Email Address is required'), 'required');
```
Comment (seamuslee001) [inline CRM/Contribute/Form/Task/Invoice.php]: @mfrausto-mdw why are you making cc and bcc required?

**Comment by eileenmcnaughton** [comment]
@mfrausto-mdw can you rebase this?

**Comment by mfrausto-mdw** [comment]
> @mfrausto-mdw can you rebase this?

Sure, will do.

**Comment by eileenmcnaughton** [comment]
@mfrausto-mdw I think the rebase went wrong - what I so is (where remote is origin)

git fetch -f origin 
git rebase -i origin/master
git push -f .....

**Comment by mfrausto-mdw** [comment]
@eileenmcnaughton Sorry for jumping back this late. I did again the rebase, let me know if look right this time, also pull request has been closed after pushing my changes, so can you re-open it?

**Comment by eileenmcnaughton** [comment]
@mfrausto-mdw yeah I can't re-open if for some reason :-(

**Comment by mfrausto-mdw** [comment]
I've created a new PR as continuity of this one:

https://github.com/civicrm/civicrm-core/pull/12970

@eileenmcnaughton

**Pull request closed without merging**